File: TP3/ex4.py
notes = []
max = int(input('Entre une note : '))
for i in range (2) :
    notes.append(int(input('Entre une note : ')))
    if notes[i] > max : 
        max = notes[i]
    i+=1
print(f"la note max est {max}")
    

File: TP3/ex4.js
document.body.innerHTML = ""
function max(){
    let max = prompt("Entre un nombre :");
    let val;
    for(let i = 0 ; i < 2 ; i++){
        val = prompt("Entre un nombre :");
        if(val > max){
            max = val;
        }
    }

    console.log("Le max est " + max);
}